stdlib/array.gr

```diff
@@ -871,3 +871,38 @@ export let slice = (startIndex, endIndex, array) => {
     init(newLength, n => array[startIndex + n])
   }
 }
+
+/**
+ * Sorts an array in-place using the quicksort algorithm.
```

Comment:
Question for the team. Do we want to specify which algorithm is used in the description of this function?

Reply:
Not here, at least.